_config.yml

```diff
@@ -32,11 +32,11 @@ redcarpet:
 
 # If true, this begins with an existing db
 use_db_cache: false
-db_cache_filename: _caches/_deploy_cache.db
-report_filename: _caches/_deploy_index_report.yaml
-report_diff_filename: _caches/_deploy_index_report_diff.yaml
-debbuild_release_path: _caches/debbuild_releases
-wiki_title_index_filename: _caches/ros_org_wiki_title_index.txt
+db_cache_filename: ../deploy/cache/_deploy_cache.db
```

Comment:
`checkout.sh` doesn't create the `cache` directory. Maybe touch a .gitignore file in the folder so it gets added in this PR?

Reply:
I rebased #7 on top of this PR and made it make the cache directory in the `build` rule.